src/layouts/Search/searchResults.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+
+import { Link } from 'gatsby'
+
+import style from './searchResults.module.css'
+
+const SearchResults = props => {
+  const { active, results, relatedResults = [] } = props
```

Comment:
Looks like best practice might say to declare default props under function declaration with `SearchResults.defaultProps`.